tutorials/resources/yolov8/yolov8_keras.py

```diff
@@ -0,0 +1,516 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
if this code was copied from ultralytics and modified, then you need to change the license. Check out the license in

> https://github.com/sony/model_optimization/blob/main/tutorials/resources/efficientdet/effdet_keras.py